src/KMonad/Keyboard/Keycode.hs

```diff
@@ -301,8 +301,525 @@ data Keycode
   | Missing253
   | Missing254
   | Missing255
-  -- Darwin
+  | Btn0
+  | BtnMisc
+  | Btn1
+  | Btn2
+  | Btn3
+  | Btn4
+  | Btn5
+  | Btn6
+  | Btn7
+  | Btn8
+  | Btn9
+  | Missing0x10a
+  | Missing0x10b
+  | Missing0x10c
+  | Missing0x10d
+  | Missing0x10e
+  | Missing0x10f
+  | BtnLeft
+  | BtnMouse
+  | BtnRight
+  | BtnMiddle
+  | BtnSide
+  | BtnExtra
+  | BtnForward
+  | BtnBack
+  | BtnTask
+  | Missing0x118
+  | Missing0x119
+  | Missing0x11a
+  | Missing0x11b
+  | Missing0x11c
+  | Missing0x11d
+  | Missing0x11e
+  | Missing0x11f
+  | BtnJoystick
+  | BtnTrigger
+  | BtnThumb
+  | BtnThumb2
+  | BtnTop
+  | BtnTop2
+  | BtnPinkie
+  | BtnBase
+  | BtnBase2
+  | BtnBase3
+  | BtnBase4
+  | BtnBase5
+  | BtnBase6
+  | Missing0x12c
+  | Missing0x12d
+  | Missing0x12e
+  | BtnDead
+  | BtnGamepad
+  | BtnA
+  | BtnB
+  | BtnC
+  | BtnX
+  | BtnY
+  | BtnZ
+  | BtnTL
+  | BtnTR
+  | BtnTL2
+  | BtnTR2
+  | BtnSelect
+  | BtnStart
+  | BtnMode
+  | BtnThumbL
+  | BtnThumbR
+  | Missing0x13f
+  | BtnDigi
+  | BtnToolPen
+  | BtnToolRubber
+  | BtnToolBrush
+  | BtnToolPencil
+  | BtnToolAirbrush
+  | BtnToolFinger
+  | BtnToolMouse
+  | BtnToolLens
+  | BtnToolQuinttap
+  | BtnStylus3
+  | BtnTouch
+  | BtnStylus
+  | BtnStylus2
+  | BtnToolDoubletap
+  | BtnToolTripletap
+  | BtnToolQuadtap
+  | BtnGearDown
+  | BtnWheel
+  | BtnGearUp
+  | Missing0x152
+  | Missing0x153
+  | Missing0x154
+  | Missing0x155
+  | Missing0x156
+  | Missing0x157
+  | Missing0x158
+  | Missing0x159
+  | Missing0x15a
+  | Missing0x15b
+  | Missing0x15c
+  | Missing0x15d
+  | Missing0x15e
+  | Missing0x15f
+  | KeyOk
+  | KeySelect
+  | KeyGoTo
+  | KeyClear
+  | KeyPower2
+  | KeyOption
+  | KeyInfo
+  | KeyTime
+  | KeyVendor
+  | KeyArchive
+  | KeyProgram
+  | KeyChannel
+  | KeyFavorites
+  | KeyEpg
+  | KeyPvr
+  | KeyMhp
+  | KeyLanguage
+  | KeyTitle
+  | KeySubtitle
+  | KeyAngle
+  | KeyZoom
+  | KeyMode
+  | KeyKeyboard
+  | KeyScreen
+  | KeyPc
+  | KeyTv
+  | KeyTv2
+  | KeyVcr
+  | KeyVcr2
+  | KeySat
+  | KeySat2
+  | KeyCd
+  | KeyTape
+  | KeyRadio
+  | KeyTuner
+  | KeyPlayer
+  | KeyText
+  | KeyDvd
+  | KeyAux
+  | KeyMp3
+  | KeyAudio
+  | KeyVideo
+  | KeyDirectory
+  | KeyList
+  | KeyMemo
+  | KeyCalendar
+  | KeyRed
+  | KeyGreen
+  | KeyYellow
+  | KeyBlue
+  | KeyChannelUp
+  | KeyChannelDown
+  | KeyFirst
+  | KeyLast
+  | KeyAb
+  | KeyNext
```

Comment:
This clashes with the alias for `KeyNextSong`, how should we remedy this?
(caught by tests in #940 (merged locally))